build/templates/_converters.py.mako

```diff
@@ -29,6 +31,18 @@ def convert_timedelta_to_microseconds(value, library_type):
     return _convert_timedelta(value, library_type, 1000000)
 
 
+def convert_iterable_to_ctypes(value, library_type=None):
```

Comment:
This is misleading.

Conversion implies that something comes in, and something else comes out. But this function returns a ctypes pointer that references the original data.

Another question: Why not use this for lists too?